scripts/lib/install-node

```diff
@@ -18,5 +19,5 @@ chown -R root:root "$n"
 chmod -R go-w "$n"
 
 # Install node and npm wrappers to /usr/local/bin
-cp "$ZULIP_PATH/../../scripts/setup/node-wrapper" /usr/local/bin/node
+cp "$NODE_BIN" /usr/local/bin/node
 cp "$ZULIP_PATH/../../scripts/setup/npm-wrapper" /usr/local/bin/npm
```

Comment:
The nvm wrapper for npm is kept intact, since npm in particular requires paths set by nvm. The penalty still remains for npm-related operations.